client/src/components/chat-header/chat-header.component.jsx

```diff
@@ -17,39 +31,81 @@ const ChatHeader = () => {
     }
   }, [currentlyActive]);
 
+  const triggerSidebar = () => {
+    dispatch({
+      type: DirectoryActionTypes.TOGGLE_SIDEBAR,
+    });
+  };
+
   return (
-    <Box boxShadow={1} py={3} px={5}>
-      <Grid container direction="row" justify="space-between" alignItems="center">
-        <Grid item>
-          <Grid container direction="row" justify="flex-start" alignItems="center" spacing={2}>
-            <Grid item>
-              <Typography>{title}</Typography>
+    <Box className={classes.header}>
+      <Box className={classes.bigPadd}>
+        <Grid container direction="row" justify="space-between" alignItems="center">
+          <Grid item xs sm>
```

Comment:
the `xs sm` isn't assigned a value so i think it's just default as `false`. do we need these added props here?

Reply:
im not exactly sure why but when i put in `xs sm` the minimum distance which the grid will snap will decrease, thus i can push the elements closer together without them snapping to 2nd row. 

With `xs sm` i could achieve 340 px without it it snaps to second row at 380 px.